src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -206,4 +211,28 @@ const PathwayVisualizer: FC = () => {
   );
 };
 
+const McodeVisualizer: FC = () => {
+  const mcode = usePatientRecords().mcodeRecords;
+  console.log(mcode);
+  type mcodeKey = keyof McodeElements & string;
+  const keysArray: mcodeKey[] = [];
+  for (const key in mcode) {
+    keysArray.push(key as mcodeKey);
+  }
+  return (
+    <table>
+      <tbody>
+        {keysArray.map(key => {
+          return (
+            <tr key={key}>
+              <td>{key}</td>
+              <td>{mcode[key] ? mcode[key] : '-'}</td>
```

Comment:
Not sure if it falls under this task or not but instead of `-` for missing data it should be the missing data link. Perhaps make a backlog item for it?

Reply:
I think it'll have to be a backlog task because there's no way to allow the user to hand select missing data beyond just selecting a transition option.

Reply:
Sounds good to me. Created https://jira.mitre.org/browse/PATHWAYS-241